File: plugins/soft-econ-reset/src/main/java/com/doceazedo/softeconreset/SoftEconReset.java
package com.doceazedo.softeconreset;

import com.doceazedo.softeconreset.commands.SoftEconResetCmd;
import org.bukkit.plugin.java.JavaPlugin;

public class SoftEconReset extends JavaPlugin {
    public static SoftEconReset instance;

    @Override
    public void onEnable() {
        instance = this;
        getCommand("soft-econ-reset").setExecutor(new SoftEconResetCmd());
    }
}

File: plugins/soft-econ-reset/src/main/java/com/doceazedo/softeconreset/SoftEconReset.kt
package com.doceazedo.softeconreset

import com.doceazedo.softeconreset.commands.SoftEconResetCmd
import com.github.shynixn.mccoroutine.bukkit.setSuspendingExecutor
import org.bukkit.plugin.java.JavaPlugin

class SoftEconReset : JavaPlugin() {
    companion object {
        lateinit var instance: SoftEconReset
    }

    override fun onEnable() {
        instance = this
        getCommand("soft-econ-reset")!!.setSuspendingExecutor(SoftEconResetCmd)
    }
}